x-pack/plugins/security_solution/public/common/components/guided_onboarding_tour/README.md

```diff
@@ -63,7 +63,7 @@ It was important that the `EuiTourStep` **anchor** is in the DOM when the tour s
         return (
           <GuidedOnboardingTourStep
             isTourAnchor={isTourAnchor}
-            step={1}
+            step={AlertsCasesTourSteps.pointToAlertName}
```

Comment:
The naming for this properties is a bit confusing. I've spent some time to understand the purpose of each. Maybe we could rename `stepId` to `securityTourId` and `step` to actual `stepId` or I misunderstood it?

Reply:
yep, it is a bit confusing to have two kinds of _steps_ here. I like the renaming Yuliia suggested. 
Also, since the component is already called `GuidedOnboardingTourStep`, maybe having `tourId` and `step` is enough?

Reply:
sure thing!